sns.tf

```diff
@@ -96,7 +96,12 @@ resource "aws_sqs_queue" "apiary_managed_logs_queue" {
       "Action": "sqs:SendMessage",
       "Resource": "arn:aws:sqs:*:*:${local.instance_alias}-s3-logs-queue",
       "Condition":{
-          "ArnEquals":{"aws:SourceArn":"arn:aws:s3:::${local.apiary_s3_logs_bucket}"}
+          "ArnEquals":{
+              "aws:SourceArn": [
+                  "arn:aws:s3:::${local.apiary_s3_logs_bucket}",
```

Comment:
you did not use `s3_logs_bucket_list` which included the apiary internal log bucket and the external log buckets.